### PR TITLE
QBJ & Memgrot

### PR DESCRIPTION
**Pull request by rjthorne**
Two packages missing from the new DB as they had nonstandard zips on the old website, so presumably the sha256 generation failed. I have rezipped and reuploaded them.

(Only noticed because I was aiming to replace all links in the descriptions to fit with the new URL format; QBJ is linked from two other packages and I couldn't find a sha256 when I tried to look it up)

### CODE REVIEW

**Comment by rjthorne** [comment]
Added minor digs07/digs08 amendments. Descriptions are 99.9% ready to go but just battling a silly bug in my json code

**Comment by rjthorne** [comment]
Meant to add, I should have actually fixed the encoding issue from the last commit now. There is an 'escape_backslash' property in  the json stringify function that has 'true' or 'false' option but actually both options enable escaping, just one does it with a backslash and one does it with a unicode backslash... there was no option to disable it for pre-escaped content, so I amended the function at source.

**Review by SpiritQuaddicted — CHANGES_REQUESTED** [review]
Thanks a lot once again!

Two minor notes with inline review comments and one here:

I'd much prefer no empty `notes` key if there are no notes. It's a fully optional field to me so it is ok if it does not exist.

File: json/by-sha256/04/04d8fa78d751ab394b54e4771f6eeef573675309825bca2928af6acff6baa105.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ },
+ "json_version": "2023.11.06-23.20",
+ "sha256": "04d8fa78d751ab394b54e4771f6eeef573675309825bca2928af6acff6baa105",
+ "tags": [
+  "author=Magnus Bj\u00f6rkbom",
+  "exceedslimits=exceeds_quake_limits",
+  "filename=memgrot.zip",
+  "game=quake",
+  "game_mode=singleplayer",
+  "link=[]()",
```
[the remark this answers is not in the document]
Comment (rjthorne) [reply json/by-sha256/04/04d8fa78d751ab394b54e4771f6eeef573675309825bca2928af6acff6baa105.json]: Done, script will now only add a link if the url field isn't empty

File: json/by-sha256/04/04d8fa78d751ab394b54e4771f6eeef573675309825bca2928af6acff6baa105.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   "timestamp": "2019-06-26T14:20:54.000Z"
+  },
+  "memgrot/memgrot.txt": {
+   "bytes": 3652,
+   "sha256": "3cba1fdd6355c0204371b7d8bd42e2dfc283f55ed1e1ba65f06534b8b91583d4",
+   "timestamp": "2019-07-07T11:21:22.000Z"
+  }
+ },
+ "install": {
+  "extract": ""
```
[the remark this answers is not in the document]
Comment (rjthorne) [reply json/by-sha256/04/04d8fa78d751ab394b54e4771f6eeef573675309825bca2928af6acff6baa105.json]: Sorry this was a silly data error, now fixed

**Comment by rjthorne** [comment]
I reverted the previous commit because I noticed encoding had gotten mangled yet again (e.g. 'Zerstu00f6rer') - fixed that problem then re-added all descriptions and notes, only this time skipping the notes property if there were no notes to add.

However... github is reporting "3,859 additions and 8 deletions" in the commit, do you know how to preview just the deletions? As as far as I know there should only be two (the empty properties from memgrot 04d8fa78d751ab394b54e4771f6eeef573675309825bca2928af6acff6baa105) and I can't find the other six.

**Comment by SpiritQuaddicted** [comment]
Awesome, thank you!

I love that you even made sure to update cross-release links like in https://www.quaddicted.com/db/v1/maps/21d33c8a494b6c569696e156f36ce43485c3ed1db4ae38f2f7722e71db83404b . That gave me a nice surprise :))

I also made sure that notes are shown, e.g. on https://www.quaddicted.com/db/v1/maps/038a48fd95fb3b128d4afd64336054110d537cca849e428ec1faf31d05a630a4

Fantastic work :heart:

**Pull request merged**